dd-sdk-android/src/main/kotlin/com/datadog/android/Datadog.kt

```diff
@@ -248,6 +248,26 @@ object Datadog {
         )
     }
 
+    /**
+     * Sets additional information on the [UserInfo] object
+     *
+     * If properties had originally been set with [Datadog.setUserInfo], they will be preserved.
+     * In the event of a conflict on key, the new property will prevail.
+     *
+     * @param extraInfo additional information. An extra information can be
+     * nested up to 8 levels deep. Keys using more than 8 levels will be sanitized by SDK.
+     */
+    @JvmStatic
+    @JvmOverloads
+    fun setExtraInfo(
```

Comment:
I'd rename this to `addUserExtraInfo`. `addExtraInfo` in itself doesn't make it explicit that it's only about the user

Reply:
will do!